packages/perseus/src/validation.types.ts

```diff
@@ -82,17 +81,26 @@ export type PerseusGradedGroupRubric = PerseusGradedGroupWidgetOptions;
 
 export type PerseusGradedGroupSetRubric = PerseusGradedGroupSetWidgetOptions;
 
-export type PerseusGrapherRubric = PerseusGrapherWidgetOptions;
+export type PerseusGrapherRubric = {
+    correct:
+        | {type: "absolute_value"; coords: [Coord, Coord]}
+        | {
+              type: "exponential";
+              asymptote: [Coord, Coord];
+              coords: [Coord, Coord];
+          }
+        | {type: "linear"; coords: [Coord, Coord]}
+        | {type: "logarithm"; asymptote: [Coord, Coord]; coords: [Coord, Coord]}
+        | {type: "quadratic"; coords: [Coord, Coord]}
+        | {type: "sinusoid"; coords: [Coord, Coord]}
+        | {type: "tangent"; coords: [Coord, Coord]};
+};
```

Comment:
I'm guessing this is copied from `PerseusGrapherWidgetOptions`, but it feels like maybe this should be a shared type between them.

```suggestion
// perseus-types.ts

// I don't know what the best name would be
type GrapherAnswerState =
        | {type: "absolute_value"; coords: [Coord, Coord]}
        | {
              type: "exponential";
              asymptote: [Coord, Coord];
              coords: [Coord, Coord];
          }
        | {type: "linear"; coords: [Coord, Coord]}
        | {type: "logarithm"; asymptote: [Coord, Coord]; coords: [Coord, Coord]}
        | {type: "quadratic"; coords: [Coord, Coord]}
        | {type: "sinusoid"; coords: [Coord, Coord]}
        | {type: "tangent"; coords: [Coord, Coord]};

export type PerseusGrapherWidgetOptions = {
    availableTypes: ReadonlyArray<
        | "absolute_value"
        | "exponential"
        | "linear"
        | "logarithm"
        | "quadratic"
        | "sinusoid"
        | "tangent"
    >;
    correct: GrapherAnswerState;
    graph: {
      // ...etc...
    }
}
    

// validation.types.ts
export type PerseusGrapherRubric = {
    correct: GrapherAnswerState;
};
```